driver.go

```diff
@@ -765,6 +768,13 @@ func (d *Driver) StartTask(cfg *drivers.TaskConfig) (*drivers.TaskHandle, *drive
 	return handle, driverNet, nil
 }
 
+// setupCgroup customizes where the cgroup lives (v2 only)
+func (d *Driver) setupCgroup(opts api.SpecGenerator) {
+	if d.cgroupV2 {
+		opts.ContainerCgroupConfig.CgroupParent = "nomad.slice"
```

Comment:
I think it's pretty safe to assume that `nomad.slice` would never change right? So we don't need to make this configurable.

Reply:
Yeah it's fine for now. I'm working on an RFC that completely overhauls Nomad's cgroup management, but that won't come about until 1.7